wonjong/services/postService.js

```diff
@@ -9,14 +8,22 @@ const createPost = async (title, content, userId) => {
 const allPostViews = async () => {
   return await postDao.allPostViews();
 };
-const patchPost = async (postId, title, content) => {
-  const createPost = await postDao.patchPost(postId, title, content);
-  return createPost;
+const patchPost = async (userId, postId, title, content) => {
+  const createPost = await postDao.patchPost(userId, postId, title, content);
```

Comment:
해당 함수의 역할이 생성이 아닌 수정 같은데, 역할에 맡게 변수명을 지어주세요.
```suggestion
  const updatePost = await postDao.patchPost(userId, postId, title, content);
```